<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>780</width>
      <height>430</height>
     </rect>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>429</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="right" native="true">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>591</width>
       <height>429</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>571</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>功能定制</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="bottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>781</width>
      <height>40</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>5</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>恢复所有默认值</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>360设置中心</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
